rosidl_generator_c/test/test_interfaces.c

```diff
@@ -896,3 +941,325 @@ int test_submessages(void)
 
   return 0;
```

Comment:
`rc` is not even defined in this function. So I think `0` is the right return code.